website/docs/reference/widgets/input.md

```diff
@@ -1,258 +1,118 @@
+| **Valid**           | Sets an expression to decide whether the user's input is considered valid. When the expression evaluates to `false`, the input is considered invalid.   |
+| **Error message**   | Sets the text of the error message to display if user input is invalid.      |
+| **Spellcheck**      | When enabled, user input is checked for spelling errors. This doesn't affect whether the input is considered invalid.   |
+| **Tooltip**         | Sets a tooltip for the widget on mouse hover. You can add hints or extra information about the required input.    |
+| **Placeholder**     | Sets the placeholder text within the input box. Use to show a hint or example value to the user.    |
+| **Show step arrows**  | Controls visibility of step arrows. Turning this toggle off hides the step arrows to increment or decrement the values in the widget. Only appears when **Data Type** is set to Number.  |
+| **Visible**         | Controls widget's visibility on the page. When turned off, the widget isn't visible when the app is published. It appears translucent when in Edit mode.     |
+| **Disabled**        | Makes the widget un-clickable or unusable. The widget remains visible to the user, but user interaction isn't allowed.     |
+| **Animate Loading** | When turned off, the widget loads without any skeletal animation. |
+| **Autofocus**       | When enabled, the user's cursor focuses on the input box automatically on page load.      |
+| [**Height**](/reference/widgets/#height)  | Configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**Fixed**: The height of the widget remains as set using drag and resize.<br/>**Auto Height**: The height of the widget reacts to content changes.<br/>  **Auto Height with limits**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
+| **Reset on Submit** | Clears the value entered by the user after the user submits with the Enter key.       |
+
+### Reference properties
+
+Reference properties are used to access the widget's data and state using code. When using reference properties, substitute `<input_name>` in the examples below with the name of your Input widget.
+
+| Reference Property | Description       | Code Snippet |
+| ---------------- | ----------------- | ------------ |
+| **text**         | Contains the widget's text / user input, either as a _string_ or _number_. depending on the widget's **Data Type** property. | `{{ Input1.text }}` |
+| **isDisabled**   | Reflects the state of the widget's **Disabled** setting.    | `{{ Input1.isDisabled }}` |
+| **isValid**      | Reflects whether the widget's input is considered **Valid**.  | `{{ Input1.isValid }}` | 
+| **isVisible**    | Reflects the state of the widget's **Visible** setting.   | `{{ Input1.isVisible }}` |
+
+### Style properties
```

Comment:
Missing - Icon

Reply:
!!!

I have no idea how I missed them, but I previously updated the Icon properties, went to the platform for reference, and couldn't find them... so I removed them. But now there they are right in front of me 🙄